west.yml

```diff
@@ -69,9 +69,9 @@ manifest:
       path: test/cmock/vendor/unity
       revision: 031f3bbe45f8adf504ca3d13e6f093869920b091
       remote: throwtheswitch
-    - name: mbedtls
-      revision: mbedtls-2.13.1
```

Comment:
@carlescufi @marti I am not sure, anymore if this is correct: are we, now moving mbedtls into modules?
If so, then we need to set the path to modules/lib/mbedtls, correct?
And also; will this be sufficient for west to checkout/update mbedtls? I.e. do we need anything else?

Reply:
@tejlmand ^^

Reply:
Didn't we talk recently about keeping two mbedTLS instances for now? One for the downstream, just as it was, and one for the upstream as a replacement for the removed `ext` copy? The latter one respecting the upstream path for the mbedTLS, i. e. `modules/crypto/mbedtls`.

Reply:
No, this version should be kept.
For time being we still need both, which were also the case before the split of zephyr mbedtls upstream.